src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -175,10 +176,10 @@
  * </p>
  *
  * <pre>
- * class Foo {
- *   int field1; // ok
- *   double field2; // ok
- *   long field3, field4 = 10L, field5; // ok
+ * class FirstClass {
+ *   int variable1; // ok
+ *   double variable2; // ok
```

Comment:
please use the same java example code in all examples.
To show user how modification of config affect behavior on same java code.

Reply:
Done. Respective changes have been made accordingly.